amundsen_application/static/js/components/common/ResourceListItem/TableListItem/index.tsx

```diff
@@ -29,16 +31,20 @@ class TableListItem extends React.Component<TableListItemProps, {}> {
       + `?index=${logging.index}&source=${logging.source}`;
   };
 
+  generateStartIconClass = (databaseId: string): string => {
+    return `icon start-icon ${getDatabaseIconClass(databaseId)}`;
+  };
+
   render() {
     const { table } = this.props;
     const hasLastUpdated = !!table.last_updated_epoch;
 
     return (
       <li className="list-group-item">
         <Link className="resource-list-item table-list-item" to={ this.getLink() }>
-          <img className="icon icon-database icon-color" />
-          <div className="content">
-            <div className="col-sm-6 col-md-8">
+          <div className="resource-info">
+            <img className={this.generateStartIconClass(table.database)} />
+            <div className="truncated">
```

Comment:
Looks like we're overusing the `truncated` class. It appears again on line 49 and 54. Same comment on UserListItem.

We can probably use a different class name and give it a flex-grow.

Reply:
Without `truncated` on this parent div, the content inside doesn't adhere to the truncation at all. However, I found that all this parent element needs is `overflow: hidden`. I will update to a new class that is just for `overflow: hidden`.
![image](https://user-images.githubusercontent.com/1790900/62815348-c86a1180-bacc-11e9-806e-8e9c52960f25.png)

Without `truncated` on the elements with the actual text that needs to get truncated, they don't render the ellipsis. I will leave these as is because it prevents the case shown below.
![image](https://user-images.githubusercontent.com/1790900/62815394-4dedc180-bacd-11e9-8900-dcd2a5ee72bf.png)